### ISSUE TITLE
Run valet automatically as background service on Windows startup

### ISSUE THREAD

**Issue by adiachenko** [vitr/valet4windows#4]
I am not sure if it's intended behavior but I need to manually launch `start.bat` file to run Valet. Of course , you can add it to `shell:startup` like any other Windows service to run on system boot but it will still open cmd popup. Is there a way to make this behavior more subtle, similarly to Mac version?

**Comment by vitr** [comment]
nope, it's unresolved issue: Caddy server couldn't be run as a service on Windows.

> To make Caddy a service on Windows, well:
> 
> If you’re a developer who wants to release a Windows Service that users can install and uninstall you should use InstallShield.

read more from @mholt
https://github.com/mholt/caddy/issues/732#issuecomment-207819773

Initially I had even two windows (1 for php and 1 for caddy :smile: ), managed to get them into just one.
We may built a service wrapper, e.g.
http://stackoverflow.com/questions/5952500/setting-a-php-script-as-a-windows-service
I'd appreciate if you have a look and share any progress on this issue.

**Comment by adiachenko** [comment]
Sorry, I wan't paying attention to unresolved issues section.

In the end, I decided to embrace this limitation and downloaded mysql and redis executables for Windows. Then I edited `start.bat` file to launch them along with caddy on `valet start`

```
start /b mysqld --console
start /b redis-server
```

If I need to run valet manually anyway, I may as well go all out for the time being and throw more portable goodies in there, lol.

P.S. My knowledge is too shallow to build a service wrapper myself.
